pxr/usd/usdLightField/schema.usda

```diff
@@ -0,0 +1,440 @@
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+        string extraIncludes = """
+            #include "pxr/usd/usdLightField/kernelBaseAPI.h"
+        """
+        token[] reflectedAPISchemas = [
+            "LightFieldKernelBaseAPI"
+        ]
+    }
+    prepend apiSchemas = [
+        "LightFieldKernelBaseAPI"
+    ]
+)
+{
+}
+
+class "LightFieldKernelGaussianSurfletAPI" (
+    doc = """Defines the gaussian surflet kernel for a given ParticleField.
+
+    The kernal shape is a circular disk centered at the origin, with the normal
+    of the plane being oriented along the positive z-axis.
+
+    The falloff function for this kernel is the gaussian falloff function,
+    where the peak of the falloff function is defined by the opacity data
+    source.
+    """
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "KernelGaussianSurfletAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+        string extraIncludes = """
+            #include "pxr/usd/usdLightField/kernelBaseAPI.h"
+        """
+        token[] reflectedAPISchemas = [
+            "LightFieldKernelBaseAPI"
+        ]
+    }
+    prepend apiSchemas = [
+        "LightFieldKernelBaseAPI"
+    ]
+)
+{
+}
+
+class "LightFieldKernelConstantSurfletAPI" (
+    doc = """Defines the constant surflet kernel for a given ParticleField.
+
+    The kernal shape is a circular disk centered at the origin, with the normal
+    of the plane being oriented along the positive z-axis.
+
+    The falloff function for this kernel is constant and the value is directly
+    defined by the opacity data source."""
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "KernelConstantSurfletAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+        string extraIncludes = """
+            #include "pxr/usd/usdLightField/kernelBaseAPI.h"
+        """
+        token[] reflectedAPISchemas = [
+            "LightFieldKernelBaseAPI"
+        ]
+    }
+    prepend apiSchemas = [
+        "LightFieldKernelBaseAPI"
+    ]
+)
+{
+}
+
+class "LightFieldRadianceBaseAPI" (
+    doc = """Defines a base-class type applied schema that all applied schema
+            that provides the a ParticleField kernel will automatically apply.
+            The purpose of this base class is to allow validation to enforce
+            that a kernel definition is present for a ParticleField"""
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "RadianceBaseAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+    }
+)
+{
+}
+
+class "LightFieldSphericalHarmonicsAttributeAPI" (
+    doc = """A ParticleField related applied schema that provides spherical
+            harmonics attributes to define the radiance of the particles.
```

Comment:
Just to be super specific, could say that the SH function is parameterized on model-space viewing angle.